tests/phpunit/Admin/WC_REST_Stripe_Settings_Controller_Test.php

```
@@ -437,6 +437,77 @@ public function test_update_settings_enforces_apple_pay_google_pay_requires_card
+		$this->get_gateway()->update_option( 'adaptive_pricing', 'no' );
+
+		$request = new WP_REST_Request( 'POST', self::SETTINGS_ROUTE );
+		$request->set_param( 'is_ap_enabled', true );
+
+		$response = $this->controller->update_settings( $request );
+
+		WC_Stripe::get_instance()->account = $original_account;
+
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertSame( 'yes', $this->get_gateway()->get_option( 'adaptive_pricing' ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_update_oc_settings_calls_maybe_reconfigure_webhooks_when_adaptive_pricing_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)